tests/console/yast2_proxy.pm

```diff
@@ -62,9 +62,6 @@ sub post_fail_hook {
 
 sub run {
     select_console 'root-console';
-    if (is_sle '15+') {
-        zypper_call('in squid');
-    }
```

Comment:
why this remove?

Reply:
because next line is to unconditionally install squid, so it's not needed.